proj/cmake/platform_macosx.cmake

```diff
@@ -114,6 +114,24 @@ source_group( "cinder\\cocoa"           FILES ${SRC_SET_COCOA} )
 source_group( "cinder\\app\\cocoa"      FILES ${SRC_SET_APP_COCOA} )
 source_group( "cinder\\audio\\cocoa"    FILES ${SRC_SET_AUDIO_COCOA} )
 
+set( MACOS_SUBFOLDER            "${CINDER_PATH}/lib/${CINDER_TARGET_SUBFOLDER}" )
+set( CINDER_STATIC_LIBS_DEPENDS "${MACOS_SUBFOLDER}/libboost_filesystem.a ${MACOS_SUBFOLDER}/libboost_system.a ${MACOS_SUBFOLDER}/libz.a" )
+
+if( NOT ( "Xcode" STREQUAL "${CMAKE_GENERATOR}" ) )
+	if(NOT CMAKE_LIBTOOL)
```

Comment:
Add comment as to why we need to use libtool instead of ar / ranlib (partially because that's what xcode does).